doc/storagedriver/azure.md

```diff
@@ -10,6 +10,7 @@ The following parameters must be used to authenticate and configure the storage
 * `accountname`: Name of the Azure Storage Account.
 * `accountkey`: Primary or Secondary Key for the Storage Account.
 * `container`: Name of the root storage container in which all registry data will be stored. Must comply the storage container name [requirements][create-container-api].
+* `domain`: (optional) Domain name for the Storage Service. Defaults to `core.windows.net`. For example, `core.chinacloudapi.cn` would be the correct value for Azure in China and `core.usgovcloudapi.net` would be the correct value for Azure Government.
```

Comment:
@stevvooe yes it's a bit hard to decide. the full API hostname is actually `https://{0}.blob.{1}` where `0`:accountname `1`:suffix. In this case user just passes `1` here as `domain` and that's not the full hostname or anything. It's just a suffix.